ui/raidboss/data/05-shb/dungeon/paglthan.js

```diff
@@ -0,0 +1,102 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+
+export default {
+  zoneId: ZoneId.Paglthan,
+  timelineFile: 'paglthan.txt',
+  timelineTriggers: [
+    {
+      // This is a rear cone attack that always follows Wide Blaster.
+      // It has a cast time of under a GCD, so we pre-warn during Wide Blaster.
+      // Only the sides are safe to call at this moment.
+      id: 'Paglthan Spike Flail',
+      regex: /Spike Flail/,
+      beforeSeconds: 4,
+      response: Responses.goSides(),
+    },
+  ],
+  triggers: [
+    {
+      id: 'Paglthan Critical Rip',
+      netRegex: NetRegexes.startsUsing({ id: '5C4E', source: 'Amhuluk' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Paglthan Electric Burst',
+      netRegex: NetRegexes.startsUsing({ id: '5C4D', source: 'Amhuluk', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paglthan Lightning Rod Gain',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'A0E' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Drop lightning outside',
```

Comment:
I don't have a particularly brief way to say this, but I think the mechanic here is to put the lightning circle on the rod so you can go back to hitting the boss. Someone can correct me if I'm wrong.

Reply:
Honestly, I didn't even see the rods outside, I was mostly focused on "where are these trust NPCs going, why are they stacked on me, PLEASE STOP, GET AWAY". I haven't had time yet to run this again, but I'll be doing that relatively soon and I'll double-check.